internal/app/apid/pkg/backend/apid.go

```diff
@@ -157,7 +165,7 @@ func (a *APID) AppendInfo(streaming bool, resp []byte) ([]byte, error) {
 	// cut off embedded message header
 	resp = resp[n1+n2:]
 	// build new embedded message header
-	prefix := append(proto.EncodeVarint((metadataField<<3)|metadataType), proto.EncodeVarint(uint64(len(resp)+len(payload)))...)
+	prefix := append(protowire.AppendVarint(nil, (metadataField<<3)|metadataType), protowire.AppendVarint(nil, uint64(len(resp)+len(payload)))...)
```

Comment:
The `DecodeVarint` and `EncodeVarint` are not implemented anywhere in `google.golang.org/protobuf`.

Reply:
just a nit, I guess we could do the following: 
```suggestion
	prefix := protowire.AppendVarint(protowire.AppendVarint(nil, (metadataField<<3)|metadataType)), uint64(len(resp)+len(payload)))
```
```